### ISSUE TITLE
get_dataset(), "The kernel appears to have died. It will restart automatically"

### ISSUE THREAD

**Issue by learsi1911** [openml/openml-python#1093]

**Comment by PGijsbers** [comment]
> We have implemented parquet support, but this is not yet in the current release.

Small correction, it should be available in the new release as soon as the production server sends valid information on where the parquet file is located.

**Comment by learsi1911** [comment]
Thank you very much for your answer, do you know approximately how long is
the time for this new version?

**Comment by PGijsbers** [comment]
@learsi1911 Could you please provide the ID of the dataset you were trying to download? And could you share how much memory was available to the kernel? That information would allow us to test whether the issue is resolved when the parquet support is fully operational.

@prabhant Do you have an estimate on when the parquet files are available from the production server?

**Comment by learsi1911** [comment]
> @learsi1911 Could you please provide the ID of the dataset you were trying to download? And could you share how much memory was available to the kernel? That information would allow us to test whether the issue is resolved when the parquet support is fully operational.
> 
> @prabhant Do you have an estimate on when the parquet files are available from the production server?

Of course the ID is 547
As I said the problem is that the first time I used "get_dataset()" I have no problem but if I try again then I get the error.

**Comment by prabhant** [comment]
The production server with parquet support will be ready in a week or two.

**Comment by mfeurer** [comment]
Dataset 547 is not really large and shouldn't result in any issues. Could you please run the failing snippet from within ipython and paste the output?

**Comment by learsi1911** [comment]
Yes, I have tried python directly in the windows console and it works,
maybe it is something related to jupyter.

**Comment by PGijsbers** [comment]
The jupyter notebook kernels typically work with much less memory than a regular python process. But as mfeurer said, the dataset isn't large and should not lead to a kernel dying. It would be helpful if you could post the code that lead to the error and the full error output.

**Comment by PGijsbers** [comment]
If the problem still occurs, please re-open this issue but provide a code example that reproduces the error.

**Comment by AstralVolt** [comment]
I have the same kind of issue. 

import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
from sklearn.datasets import make_classification
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler
from sklearn.linear_model import LogisticRegression
from sklearn.metrics import accuracy_score, confusion_matrix, classification_report
import openml

from IPython.display import Image, display

credit_data = openml.datasets.get_dataset(31) 
X, y, _, _ = credit_data.get_data(target=credit_data.default_target_attribute)

The kernel dies and restarts after this. Alternatively I tried writing this in the last line aswell

X, y, _, _ = credit_data.get_data(target=credit_data.default_target_attribute, dataset_format="dataframe")

Still with no luck. Its not fixing. Tried possibly everything including upgrading openml

**Comment by PGijsbers** [comment]
Could you let us know which OS you are using, as well as the python version and a list of installed dependencies? (`python -V`  and `python -m pip list`)

**Comment by SubhadityaMukherjee** [comment]
Closing since it's stale and it does seem to work on colab and local jupyter notebook.
